Analysis/DEEP2_R23_O32/Plotting/te_metal_plots.py

```diff
@@ -305,16 +356,12 @@ def Jiang_comparison(fitspath, a, b, c, d, e):
     pdf_pages.close()
 
 
-def dm_Jiang_comparison():
-    fitspath_ini = '/Users/reagenleimbach/Desktop/Zcalbase_gal/'
+def dm_jiang_comparison(fitspath, pdf_name):
+    """
```

Comment:
Some additional information might be needed here.  Not sure what "dm" stands up.

Reply:
After reviewing, these functions were not complete. I suspect that I wrote them while we were writing the paper and didn't finish them because we decided we didn't need what they were going to produce. I will archive these two functions. 
I believe "dm" stands for direct metallicity because I was making a comparison graph.